CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Breaking changes:
 Features:
 
 Fixes:
+- [#2319](https://github.com/rails-api/active_model_serializers/pull/2319)
+  Fix Rails 6.0 deprication warnings and update test fixture schema to use `timestamps` instead of `timestamp`
+  Fixes #2319. (@kylekeesling)
```

Comment:
@kylekeesling you're almost done, but I guess this PR would fix #2316, no? :smile:

Reply:
lol - oops - I guess I fixed my own PR! Changing now.